db/index.js

```diff
@@ -61,6 +61,18 @@ const updateComment = (comment_id, content) => Comment.updateOne({ comment_id },
 
 const deleteComment = (comment_id) => Comment.deleteOne({ comment_id });
 
+const dropDB = () => db.dropDatabase();
+
+// eslint-disable-next-line consistent-return
+const insertMany = (comments) => Comment.insertMany(comments, (err, docs) => {
+  if (err) {
+    console.log(err);
+    return err;
+  }
+  console.log(JSON.stringify(docs.length));
+  return (docs.length);
+});
+
```

Comment:
Nice!✓ maybe just remove lines 21-23.

Reply:
Removed from the next branch. Good catch.